.github/auto-label.json

```diff
@@ -1,5 +1,6 @@
 {
   "rules": {
-    "migrations": ["**/migrations/**/*"]
+    "migrations": ["**/migrations/**/*"],
+    "🧭 project: implementation plan": ["**/*implementation_plan*"]
```

Comment:
Do you think we should also have a similar one for the project proposal label?

Reply:
Yes! Added in 0cf8668.